### ISSUE TITLE
What is the variable named 'vuv' ?

### ISSUE THREAD

**Issue by ayush1801** [pc2752/ss_synthesis#3]
Hi @pc2752 ! It would be very helpful if you could tell me what is the meaning/ significance of the 'vuv' variable. In your paper, I didn't find any information regarding this variable. One of the appearance of this variable is in :

vuv = tf.layers.dense(vuv, 1, activation=tf.nn.sigmoid)  in nr_wavenet function.

Also, please enlighten about the significance of MFSCs ( Mel Frequency Spectral Coefficients ) rather than using the MFCCs. I am not able to find any literature regarding MFSCs. Thanks in Advance!

Best Regards,
Ayush Agrawal
